lib/core/src/Cardano/Wallet/DB/Store/Wallets/Model.hs

```diff
@@ -77,13 +76,19 @@ import Fmt
     ( Buildable, build )
 
 import qualified Cardano.Wallet.DB.Sqlite.Schema as DB
+import Cardano.Wallet.DB.Store.CBOR.Model
```

Comment:
Could you regroup the imports without `qualified` into the other section — to follow our coding standards for imports?

https://input-output-hk.github.io/adrestia/code/Coding-Standards#stylish-haskell-is-used-to-format-grouped-imports--language-pragmas